db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1515,6 +1515,13 @@ class NonBatchedOpsStressTest : public StressTest {
         }
       }
 
+      if (ro_copy.allow_unprepared_value) {
+        if (!iter->PrepareValue()) {
+          s = iter->status();
```

Comment:
Will this `s` possibly be Status::OK() so we end up not flagging the case where iter->PrepareValue() failed??